packages/kbn-es-query/src/filters/helpers/meta_filter.ts

```diff
@@ -113,11 +113,7 @@ export const unpinFilter = (filter: Filter) =>
  * @public
  */
 export const isFilter = (x: unknown): x is Filter =>
-  !!x &&
-  typeof x === 'object' &&
-  !!(x as Filter).meta &&
-  typeof (x as Filter).meta === 'object' &&
-  typeof (x as Filter).meta.disabled === 'boolean';
+  !!x && typeof x === 'object' && !!(x as Filter).meta && typeof (x as Filter).meta === 'object';
```

Comment:
Discussed with @Dosant and, since `meta.disabled` is an optional property, there is no need to check whether or not `disabled` is a boolean value because it is allowed to be `undefined`.